extensions/expose/init.lua

```diff
@@ -318,11 +321,11 @@ local function getHints(self,windows)
     end
   end
   local function normalize(t,n) --change in place
-    local _
     while #t>0 and tlen(t.apps)>0 do
       if n>self.ui.maxHintLetters or (tlen(t.apps)==1 and n>1 and not hasSubHints(t))  then
         -- last app remaining for this hint; give it digits
-        local app=next(t.apps)
```

Comment:
I haven't checked this in detail yet, but I'm wondering if the `next()` call is important here even if `app` isn't used?

Reply:
@cmsj - Good point - that makes sense. It should probably just be `next(t.apps)` without the `local app`?

Reply:
That would be my expectation, yes.

Reply:
Fixed in my branch.